python/cugraph-pyg/cugraph_pyg/tests/test_cugraph_store.py

```diff
@@ -383,16 +371,22 @@ def test_get_tensor_from_tensor_attrs(graph):
         assert cugraph_store.get_tensor(tensor_attr).tolist() == data.tolist()
 
 
-@pytest.mark.skip(
-    "Skipping for now, unskip after https://github.com/rapidsai/cugraph/pull/3289"
-)
+@pytest.mark.skipif(isinstance(torch, MissingModule), reason="torch not available")
 def test_get_tensor_size(graph):
     F, G, N = graph
-    cugraph_store = CuGraphStore(F, G, N, backend="cupy")
+    cugraph_store = CuGraphStore(F, G, N)
 
     tensor_attrs = cugraph_store.get_all_tensor_attrs()
     for tensor_attr in tensor_attrs:
         sz = N[tensor_attr.group_name]
 
         tensor_attr.index = np.arange(sz)
-        assert cugraph_store.get_tensor_size(tensor_attr) == sz
+        assert cugraph_store.get_tensor_size(tensor_attr) == torch.Size((sz,))
+
+
+def test_serialize(multi_edge_multi_vertex_no_graph_1):
+    import pickle
+
+    F, G, N = multi_edge_multi_vertex_no_graph_1
+    cugraph_store = CuGraphStore(F, G, N)
+    pickle.dumps(cugraph_store)
```

Comment:
should this at least have an assert, maybe something like this (assuming `==` works here):
```suggestion
    assert pickle.loads(pickle.dumps(cugraph_store)) == cugraph_store
```